File: RADWIMPS.dart
class RADWIMPS {
  String zennzennzennse = "";

  RADWIMPS then() {
    zennzennzennse += "前";
    return this;
  }

  RADWIMPS 世() {
    zennzennzennse += "世";
    print(zennzennzennse);
    return this;
  }
}

void main() {
  RADWIMPS().then().then().then().世();
}

File: RADWIMPS.nim
type RADWIMPS = object
const 君の: RADWIMPS = RADWIMPS()

method then(value: RADWIMPS): RADWIMPS =
  stdout.write "前"
  return value

method 世(value: RADWIMPS): void =
  echo "世"

君の.then.then.then.世
